internal/bgp/frr/frr.go

```diff
@@ -168,9 +168,13 @@ func (sm *sessionManager) createConfig() (*frrConfig, error) {
 
 	config := &frrConfig{
 		Hostname: hostname,
-		Loglevel: "informational", // TODO - make loglevel configurable via envvar.
+		Loglevel: "informational",
 		Routers:  make(map[string]*routerConfig),
 	}
+	frrLogLevel, found := os.LookupEnv("FRR_LOGGING_LEVEL")
```

Comment:
How about we prefix all metallb env vars with `METALLB_` ?

Reply:
currently all FRR env started with FRR_blah...u suggesting adding METALLB_FRR_balh....to new and existing envvar ?

Reply:
That would be my preference, but I forgot that we had other vars as `FRR_` already, so don't worry about it